src/utils/git.ts

```diff
@@ -5,7 +5,9 @@ export async function listFiles(): Promise<string[]> {
   core.startGroup('Listing all files tracked by git')
   let output = ''
   try {
-    output = (await exec('git', ['ls-files', '-z'])).stdout
+    const execOptions = {silent: true} as const // On large repos, this output can be gigantic
+    output = (await exec('git', ['ls-files', '-z'], execOptions)).stdout
+    console.log(output.slice(0, 600))
```

Comment:
Just need to still figure out that this doesn't stop this from actually outputting anything. I want to prevent it spamming the Github Actions logs.